### DISCUSSION TITLE
Carbon API support?

### DISCUSSION THREAD

**Discussion by jjuran** [ravynsoft/ravynos#308] [Ideas]
Is it a goal to support Carbon-based applications by implementing (parts of) Carbon.framework (or its subframeworks, like ApplicationServices)?  If so, which APIs are of the highest priority?

**Comment by mszoek** [comment]
Hi! Supporting Carbon itself is not a goal, but a few pieces such as HIToolbox are still used by Cocoa apps. We need to implement those bits.
